model/host/host.go

```diff
@@ -388,79 +388,46 @@ type SleepScheduleInfo struct {
 	IsBetaTester bool `bson:"is_beta_tester,omitempty" json:"is_beta_tester,omitempty"`
 }
 
-// Validate checks that the sleep schedule provided by the user is valid.
-func (i *SleepScheduleInfo) Validate() error {
```

Comment:
This file is exactly the same as it was before, but just slightly refactored. There's a new constructor `NewSleepScheduleInfo`, a new options struct for the options for users to set a sleep schedule, and `Validate` has been cut/pasted into the options struct below.

Unfortunately, git represents this refactor as a huge diff, even though little has changed 🤷